Updated deprecated links in incoming-webhooks _index.md (#879)
mattermost/mattermost-developer-documentation: site/content/integrate/incoming-webhooks/_index.md

| Parameter | Description | Required |
|---|---|---|

> | text | [Markdown-formatted](https://docs.mattermost.com/messaging/formatting-text.html) message to display in the post.<br> To trigger notifications, use `@<username>`, `@channel` and `@here` like you would in normal Mattermost messaging. | If `attachments` is not set, yes |

Reviewer: ```suggestion
| text | [Markdown-formatted](https://docs.mattermost.com/messaging/formatting-text.html) message to display in the post.<br> To trigger notifications, use `@<username>`, `@channel`, and `@here` like you would in normal Mattermost messaging. | If `attachments` is not set, yes |
```

> | channel | Overrides the channel the message posts in. Use the channel's name and not the display name, e.g. use `town-square`, not `Town Square`.<br> Use an "@" followed by a username to send to a direct message.<br> Defaults to the channel set during webhook creation.<br> The webhook can post to any public channel and private channel the webhook creator is in.<br> Posts to direct messages will appear in the DM between the targeted user and the webhook creator. | No |

Reviewer: ```suggestion
| channel | Overrides the channel the message posts in. Use the channel's name and not the display name, e.g. use `town-square`, not `Town Square`.<br> Use an "@" followed by a username to send to a Direct Message.<br> Defaults to the channel set during webhook creation.<br> The webhook can post to any Public channel and Private channel the webhook creator is in.<br> Posts to Direct Messages will appear in the Direct Message between the targeted user and the webhook creator. | No |
```

| icon\_url | Overrides the profile picture the message posts with.<br> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
| icon\_emoji | Overrides the profile picture and `icon_url` parameter.<br> Defaults to none and is not set during webhook creation.<br> Must be enabled [in the configuration](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons).<br> The expected content is an emoji name, as typed in a message, either with or without `:`. | No |
| attachments | [Message attachments](https://docs.mattermost.com/developer/message-attachments.html) used for richer formatting options. | If `text` is not set, yes |
| type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with "custom\_". | No |

> | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: "from\_webhook", "override\_username", "override\_icon\_url", "override\_icon\_emoji", "webhook\_display\_name", "card", and "attachments".<br> Props `card` allows for extra information (markdown formatted text) to be sent to Mattermost that will only be displayed in the RHS panel after a user clicks on an 'info' icon displayed alongside the post.<br> The info icon cannot be customized and is only rendered visible to the user if there is `card` data passed into the message.<br> This is only available in v5.14+.<br> There is currently no Mobile support for `card` functionality. | No |

Reviewer: ```suggestion
| props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: "from\_webhook", "override\_username", "override\_icon\_url", "override\_icon\_emoji", "webhook\_display\_name", "card", and "attachments".<br> Props `card` allows for extra information (Markdown-formatted text) to be sent to Mattermost that will only be displayed in the RHS panel after a user selects the **info** icon displayed alongside the post.<br> The **info** icon cannot be customized and is only rendered visible to the user if there is `card` data passed into the message.<br> This is only available in v5.14+.<br> There is currently no Mobile support for `card` functionality. | No |
```
